node_scripts/install/spark.py

```diff
@@ -88,12 +88,14 @@ def start_history_server():
     # configure the history server
     spark_event_log_enabled_key = 'spark.eventLog.enabled'
     spark_event_log_directory_key = 'spark.eventLog.dir'
+    spark_history_fs_log_directory = 'spark.history.fs.logDirectory'
     path_to_spark_defaults_conf = os.path.join(spark_home, 'conf/spark-defaults.conf')
     properties = parse_configuration_file(path_to_spark_defaults_conf)
+    required_keys = [spark_event_log_directory_key, spark_history_fs_log_directory]
```

Comment:
Missing spark_event_log_enabled?

I think this broke between Spark versions... Different Spark versions have different requirements, which makes this a difficult dictionary to maintain.